src/main/java/com/iexec/worker/executor/TaskManagerService.java

```diff
@@ -270,7 +287,13 @@ ReplicateActionResponse compute(String chainTaskId) {
             return ReplicateActionResponse.failureWithStdout(cause,
                     postResponse.getStdout());
         }
-        return ReplicateActionResponse.successWithStdout(appResponse.getStdout());
+        return ReplicateActionResponse.successWithLogs(
+                new ComputeLogs(
+                        workerConfigurationService.getWorkerWalletAddress(),
```

Comment:
Not sure we should "trust" worker when setting `walletAddress` himself, what do you think?

Reply:
I see what you mean. Let's remove this information from worker-side. This will be the responsibility of the Scheduler to keep it up-to-date.
Please see [Worker commit](https://github.com/iExecBlockchainComputing/iexec-worker/pull/448/commits/d105ce3587a8a40c4f1646fbca6f8815e2acf9cd) and [Core commit](https://github.com/iExecBlockchainComputing/iexec-core/pull/504/commits/9516931a40362172610553b3682a2c213e107dd5).

Reply:
Checked, thanks :)